src/main/java/frc/robot/subsystems/Drive/DriveSubsystem.java

```diff
@@ -320,9 +320,11 @@ public void periodic() {
 
      }
      public double calculateArrivalSpeedWithXPID(double distanceX){
+          this.PIDControllerArriveX.setSetpoint(distanceX);
```

Comment:
you need to create a function to set its setpoint you dont need to integreate it in this function just create a function called 
setSetpointArrivalX and setSetpointArrivalY